README.md

```diff
@@ -53,8 +53,25 @@ the open source community.
 
 ## Quick Start Guide
 
-The easiest way to deploy Volcano is to use the Helm chart.
+The easiest way to deploy Volcano is to use the Helm chart.  Volcano can be installed by cloning new code and also without cloning code.
 
+## Without Cloning Code
+### Helm Charts
+
+Add Helm repo using following command,
+
+```
+helm repo add volcano https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/helm_package
```

Comment:
let's have a repo for that.

Reply:
sure... That'll be better
@k82cn can you create a repo under volcano-sh?

Reply:
And if possible could you enable github pages for that repo?